tests/acceptance/features/apiGraph/fullSearch.feature

```diff
@@ -29,7 +29,7 @@ Feature: full text search
       | spaces           |
 
 
-  Scenario Outline: sharee searches files using a tag
+  Scenario Outline: Sharee tags another file from a shared resource using the same tag as the sharer
```

Comment:
we don't start scenarios with capital letters
```suggestion
  Scenario Outline: sharee tags another file from a shared resource using the same tag as the sharer
```

Reply:
Oh soory i miss that